### ISSUE TITLE
Cannot assign to read only property 'data' of object '#<Object>'

### ISSUE THREAD

**Issue by mistre83** [yang-wei/rd3#49]
I'm trying to use `LineChart` with Dates and i've just copied [this](https://yang-wei.github.io/rd3/docs/new/date.html) example but i get the error on issue title.

This is a jsfiddle example: https://jsfiddle.net/cnomx3w1/

**Comment by yang-wei** [comment]
Unlike [AreaChart](https://github.com/yang-wei/rd3/blob/master/src/areachart/AreaChart.jsx#L70), we do not process date object in LineChart (which is not consistent)

Would you like to give it a try ?

**Comment by mistre83** [comment]
Yes, I can evaluate to use AreaChart instead of LineChart. At the end, it have all information i need.

Just to be sure, i have a set of measurements and this can be registered by hour, so i will have to use "hour" as tick interval on the X axis. For the Y axis, i dont have a fixed "cap".

For example, one chart can have a cap of 10k, and another can have a cap to 1k. I will use this as "generic" container, so i cant provide a cap.

How i can configure AreaChart to have this behaviour ?

In xAxisTickInterval i can say:

`xAxisTickInterval={{unit: 'hour', interval: 2}}` ?
